index.html

```diff
@@ -259,7 +259,7 @@ <h3 class="service_title">ACM Quest</h3>
 
     <!--=========== BEGIN WORKS SECTION ================-->
     <section id="works">
-      <!-- BEGIN MILESTONE WORSK SECTION -->
+      <!-- BEGIN MILESTONE WORKS SECTION -->

```

Comment:
Ahah, I told you, there are many of them. So, first we need to file a bug on them and correct them all at a time !! Instead spend you time on some productive work !! :)